### PR TITLE
Fix #165624 check-host.net

### PR DESCRIPTION
**Pull request by xglossyntx**
# Creating the pull request

> Please include a summary of the change and which issue is fixed\
> If the related issue does not exist in our repository, please create it before making pull request\
> It is highly recommended to use our [Web Reporting Tool](https://kb.adguard.com/technical-support/reporting-tool) instead of creating an issue on GitHub directly\
> Please note, that we verify every pull request manually, so it may take time to apply it

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

check-host.net: <https://github.com/AdguardTeam/AdguardFilters/issues/165624>

#### If possible, a screenshot of a page or application should not be cropped too much. Otherwise, it is not always clear where the element is located

0. **DO NOT** upload screenshots with sexually explicit material on GitHub directly.\
 Instead, upload it to a third-party image hosting and post the link to it without preview (!) here.\
 Also, mention if the link leads to NSFW content;

1. Add screenshots of the problem. You can drag and drop images or paste them from clipboard

    Use `<details> </details>` tag to hide screenshots under the spoiler;

2. Describe the issue in detail until it is absolutely clear from the screenshot what the problem is

    You can also indicate any other information that you think the developers should know.

**Warning:** Please remove personal information before uploading screenshots!

1. Your comment

2. Screenshots

<details>

<summary>Screenshot 1:</summary>

</details>

<details>

<summary>Screenshot N:</summary>

</details>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Review by zloyden — COMMENTED** [review]
It was fixed directly.

**Comment by xglossyntx** [comment]
@zloyden Was one of the rules not working for you? I'm still somewhat new to this, so if something was broken, I would like to know so I don't end up doing it again.

**Comment by zloyden** [comment]
Actually, they work fine on my end. I'll comment on your rules.

**Comment by zloyden** [comment]
> I'm still somewhat new to this

Btw, congratulations on opening a pull request.

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -326,9 +326,10 @@ lovethemaldives.com##div[class*="-rectangle-"]
 allnovascotia.com##div[style="height:130px;text-align:center;background-color:#eef2f5"]
 tutlehd.xyz###overlayer
 portableapps.com##.ez-sidebar-wall
-check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯)$/):upward(2)
-check-host.net###hostign
-/^https:\/\/check-host\.net\/images\/[0-9]?_?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png\?[0-9]+$/$domain=check-host.net
+check-host.net#?#div + div[class*=" "]:last-child:contains(/^(χ|Χ|х|𝕏|🗙|🗶|🗴|✘|✗|⨯|☓|×|✕|✖|X|x)$/):upward(2)
+check-host.net###hostigm
+/^https:\/\/check-host\.net\/images\/[a-z]+[0-9]?_?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png\?[0-9]+$/$domain=check-host.net
```
Comment (zloyden) [inline BaseFilter/sections/specific.txt:331]: `images\/[a-z]+` it's better to point that the letter was one or not.
Perhaps regex rule is not needed at all as it's covered by the `#?#` rule.

File: BaseFilter/sections/specific.txt [hunk 2]
```diff
@@ -326,9 +326,10 @@ lovethemaldives.com##div[class*="-rectangle-"]
...
+check-host.net##a[href^="https://specialrdp.com/"]
```
Comment (zloyden) [inline BaseFilter/sections/specific.txt:332]: Not needed as you have `check-host.net###hostigm`.

File: BaseFilter/sections/specific.txt [hunk 3]
```diff
@@ -326,9 +326,10 @@ lovethemaldives.com##div[class*="-rectangle-"]
 allnovascotia.com##div[style="height:130px;text-align:center;background-color:#eef2f5"]
 tutlehd.xyz###overlayer
 portableapps.com##.ez-sidebar-wall
-check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯)$/):upward(2)
-check-host.net###hostign
-/^https:\/\/check-host\.net\/images\/[0-9]?_?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png\?[0-9]+$/$domain=check-host.net
+check-host.net#?#div + div[class*=" "]:last-child:contains(/^(χ|Χ|х|𝕏|🗙|🗶|🗴|✘|✗|⨯|☓|×|✕|✖|X|x)$/):upward(2)
```
Comment (zloyden) [inline BaseFilter/sections/specific.txt:329]: New `x` values better add to the end or at the beginning. No need to add all the values, just the ones they use at the moment.

File: BaseFilter/sections/specific.txt (hunk 2 above)
Comment (xglossyntx) [reply BaseFilter/sections/specific.txt:332]: It was backup rule do to them always updating the site

File: BaseFilter/sections/specific.txt (hunk 3 above)
Comment (xglossyntx) [reply BaseFilter/sections/specific.txt:329]: Where are you seeing all these? `check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯|✗)$/):upward(2)`  The only one I can find is `✗` . They did, however, use them before, but I can't seem to find anything in their HTML code right now that indicates there still using them.

Comment (xglossyntx) [reply BaseFilter/sections/specific.txt:329]: Never mind they just updated now its a `+`

File: BaseFilter/sections/specific.txt (hunk 1 above)
Comment (xglossyntx) [reply BaseFilter/sections/specific.txt:331]: Thanks. I was not really sure about that rule, but it did seem to be working and was not breaking anything, so I let it be.

**Pull request closed without merging**